### PR TITLE
Redesign vote reading screen

### PR DESCRIPTION
**Pull request by harrygr**

### CODE REVIEW

File: src/pages/ViewVotes/index.tsx [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+import { VoteCarousel } from './VoteCarousel';
 
 export interface Props extends RouteComponentProps<{ id: string }> {
   analytics: Analytics;
 }
 
 interface State {
-  currentDotdVote: number;
-  currentMotmVote: number;
+  type: 'motm' | 'dotd';
```
Comment (Jarrydcrawford) [inline src/pages/ViewVotes/index.tsx]: Worth pulling these out to a new union type?

File: src/components/BinaryToggle.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+const Option = ({ id, value, children, onChange, checked }: OptionProps) => {
+  return (
+    <OptionContainer>
+      <input
+        type="radio"
+        value={value}
+        checked={checked}
+        name={name}
+        onChange={() => onChange(value)}
```
Comment (Jarrydcrawford) [inline src/components/BinaryToggle.tsx]: Does this work if you pass in the onChange as a reference? 🤔

File: src/pages/ViewVotes/index.tsx [hunk 3]
```diff
@@ -38,125 +41,75 @@ class ContainedViewVotes extends React.Component<Props, State> {
...
+                          value="dotd"
+                          id="toggle-dotd"
+                          onChange={setType}
+                          checked={this.state.type === 'dotd'}
+                        >
+                          DoD
+                        </BinaryToggle.Option>
+                      </BinaryToggle.Container>
+
+                      {/* Setting the key forces a rerender when the vote type changes */}
```
Comment (Jarrydcrawford) [inline src/pages/ViewVotes/index.tsx]: If you wanna keep this then cool, otherwise no need.

File: src/pages/ViewVotes/VoteCarousel.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  box-shadow: ${shadows.raised};
+
+  > div {
+    padding: ${spacing.goat};
+  }
+
+  > footer {
+    padding: ${spacing.goat} ${spacing.whale};
+    position: relative;
+  }
```
Comment (Jarrydcrawford) [inline src/pages/ViewVotes/VoteCarousel.tsx]: I'm almost tempted to say pull these out into their own styled component but there's very little here to warrant it. 😬

File: src/components/BinaryToggle.tsx (hunk 2 above)
Comment (harrygr) [reply src/components/BinaryToggle.tsx]: As in pre-loaded with the value? Can you give an example?

File: src/pages/ViewVotes/index.tsx (hunk 3 above)
Comment (harrygr) [reply src/pages/ViewVotes/index.tsx]: If the key is omitted the component throws an exception when the contents change due to the way the Flikity works. The comment helps explain why the key is there.

File: src/pages/ViewVotes/VoteCarousel.tsx (hunk 4 above)
Comment (harrygr) [reply src/pages/ViewVotes/VoteCarousel.tsx]: Yeah thought about compound components. If it gets any more complex then sure, but for now I think it's simple enough.

File: src/pages/ViewVotes/index.tsx (hunk 3 above)
Comment (Jarrydcrawford) [reply src/pages/ViewVotes/index.tsx]: Yeah fair enough, was going to suggest linking to the React blog article that explains this but couldn't find it so 🤷‍♂️

File: src/components/BinaryToggle.tsx (hunk 2 above)
Comment (Jarrydcrawford) [reply src/components/BinaryToggle.tsx]: I thought that the React onChange gives back the value as the callback result, I could definitely be wrong, so like 

```tsx
onChange={onChange}
```

Comment (harrygr) [reply src/components/BinaryToggle.tsx]: It gives the event, so yeah I guess we could do that. We'd need to change the type of the the callback in props though. But that would probably be nice.

**Pull request merged**